dnsimple/dnsimple.go

```diff
@@ -327,6 +327,15 @@ type ErrorResponse struct {
 	AttributeErrors map[string][]string `json:"errors"`
 }
 
+// An alternate type of ErrorResponse, used internally.
+type InternalAltErrorResponse struct {
```

Comment:
Is it expected to be used outside this package, or it is "internal" as if just used internally? In the second case, we should not export it (aka keep the `internal` lowercase instead of `Internal`).

Reply:
Thanks for the suggestion! It is just used internally so I have made the fix here: https://github.com/dnsimple/dnsimple-go/pull/191/commits/b5e8efdaeb842029b0bca49f9e737d7f553bdc73